applications/StructuralMechanicsApplication/python_scripts/automatic_rayleigh_parameters_computation_process.py

```diff
@@ -118,7 +118,9 @@ def ExecuteBeforeSolutionLoop(self):
                 eigen_linear_solver = eigen_solver_factory.ConstructSolver(self.settings["eigen_system_settings"])
                 builder_and_solver = KM.ResidualBasedBlockBuilderAndSolver(eigen_linear_solver)
                 eigen_scheme = SMA.EigensolverDynamicScheme()
-                eigen_solver = SMA.EigensolverStrategy(self.main_model_part, eigen_scheme, builder_and_solver)
+                eigen_solver = SMA.EigensolverStrategy(self.main_model_part, eigen_scheme, builder_and_solver,
```

Comment:
:+1:

Reply:
sorry that my fault I forgot to update this